app/helpers/message_format_helper.rb

```diff
@@ -2,7 +2,11 @@ module MessageFormatHelper
   include RegexHelper
 
   def transform_user_mention_content(message_content)
-    message_content.gsub(MENTION_REGEX, '\1')
+    if message_content
```

Comment:
Could you solve this using the `ternary` operator? 
`message_content.presence ?  regex_value : ''`

Reply:
done, haha, personally think `if-else` is easier to understand than `ternary`